storage/register.go

```diff
@@ -0,0 +1,20 @@
+package storage
+
+import (
+	"github.com/onflow/flow-go/model/flow"
+	"io"
+)
+
+type Register interface {
+	RegisterReader
+	RegisterWriter
+	io.Closer
+}
+
+type RegisterReader interface {
+	Get(ID flow.RegisterID, height uint64) flow.RegisterEntry
```

Comment:
```suggestion
	Get(ID flow.RegisterID, height uint64) (flow.RegisterEntry, error)
```

Please document what error will return when register is not found

Reply:
Yep will add comments, sorry.